### DISCUSSION TITLE
Update destroy() method for paginator plugin

### DISCUSSION THREAD

**Discussion by katyapavlenko** [salesforce/akita#631] [Show and tell]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Request

It would be good to have a possibility to destroy paginator and complete all subjects inside of it to prevent memory leaking.
Looks like `this.page` should be completed in destroy() method 
https://github.com/datorama/akita/blob/master/libs/akita/src/lib/plugins/paginator/paginatorPlugin.ts#L69

**Comment by NetanelBasal** [comment]
You're the one that subscribes to it. You are responsible for unsubscribing.
